packages/mjml-section/README.md

```diff
@@ -52,3 +52,4 @@ padding-left        | px          | section left offset            | n/a
 padding-right       | px          | section right offset           | n/a
 padding-top         | px          | section top offset             | n/a
 text-align          | string      | css text-align                 | center
+vertical-align      | string      | css vertical-align             | top
```

Comment:
this had been removed intentionally so it shouldn't be re-added

Reply:
commit 3b9a67e1